.gitignore

```diff
@@ -50,3 +50,4 @@ app.*.map.json
 /android/app/debug
 /android/app/profile
 /android/app/release
+macos/Podfile.lock
```

Comment:
The `Podfile.lock` is supposed be under version control, since it ensures that everyone building the project uses the same versions of the libraries. This specific file only matters for the macOS version, which isn't a version we're targeting anyway. I'm unclear why it's being added to `.gitignore`.

Reply:
Great I re-added the podfile.lock and removed it from the gitignore